cirq-core/cirq/transformers/gauge_compiling/gauge_compiling_test_utils_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pytest
+import numpy as np
+import cirq
+from cirq.transformers.gauge_compiling.gauge_compiling_test_utils import GaugeTester
+from cirq.transformers import GaugeTransformer, GaugeSelector, ConstantGauge
+
+
+class _TEST_GATE(cirq.testing.TwoQubitGate):
+    unitary = cirq.unitary(cirq.CZ**0.123)
+
+    def _unitary_(self) -> np.ndarray:
+        return self.unitary
+
+
+_TEST_TARGET = _TEST_GATE()
```

Comment:
for sake of consistency - please rename `_TEST_TARGET` --> `_EXAMPLE_TARGET`.

Reply:
done